docs/release-notes/early-stopping-bug.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fixes**
+
+-  Distributed training: Fix a bug where a distributed training trial that calls
+   ``context.set_stop_requested`` caused the trial to error instead of successfully complete.
```

Comment:
-  Distributed training: We fixed a bug where a distributed training trial that calls
   ``context.set_stop_requested`` was causing the trial to error and preventing it from completing successfully.